tests/integration_tests/databases/api_tests.py

```diff
@@ -686,6 +686,21 @@ def test_if_ssh_tunneling_flag_is_not_active_it_raises_new_exception(
         # the DB should not be created
         assert model is None
 
+    def test_get_table_details_with_slash_in_name(self):
+        table_name = "table_with/slash"
+        database = get_example_database()
+        if database.backend in ("mysql", "sqlite"):
```

Comment:
It seems [MySQL](https://stackoverflow.com/questions/9801760/using-slash-in-mysql-database) and [SQLite](https://stackoverflow.com/questions/3694276/what-are-valid-table-names-in-sqlite) support table names with slashes but you need to escape them.